### ISSUE TITLE
Alturos.Yolo on Windows 10

### ISSUE THREAD

**Issue by EstherHD** [AlturosDestinations/Alturos.Yolo#17]
Hi,
I try to run Alturos.Yolo on 
- Windows 10 
- VS2017
- Cuda V10.0 + cudnn
and get the exception  Unable to load DLL x64\yolo_cpp_dll_gpu.dll
The yolo_cpp_dll_cpu.dll is running without problems.
Does anyone know this problem? Do I need another yolo_cpp_dll_gpu.dll compiled with cuda V10.0?

Thanks a lot

**Comment by tinohager** [comment]
I currently use cuda 9.2 you must compile self if you nedd cuda 10

**Comment by EstherHD** [comment]
Hi Tino,thanks for the fast answer.
I hope I can do it ;-)
ThanksEsther

    Am Mittwoch, 14. November 2018, 16:30:33 MEZ hat Tino Hager <notifications@github.com> Folgendes geschrieben:  
 
 
I currently use cuda 9.2 you must compile self if you nedd cuda 10

**Comment by purnadika** [comment]
Hi @EstherHD 
Could you please share your work for win10?
I'm using win10 too
Having problem like this
![image](https://user-images.githubusercontent.com/9426610/48963852-92344280-efce-11e8-81b0-e8a2b269cfc1.png)

**Comment by purnadika** [comment]
oops. Fixed by comment out this
![image](https://user-images.githubusercontent.com/9426610/48964391-c6146580-efd8-11e8-8938-fad246573951.png)

**Comment by tinohager** [comment]
@purnadika have you install c++ 2017 redistributable?

**Comment by tinohager** [comment]
@purnadika I have push a new Version to nuget can you check again?

**Comment by tinohager** [comment]
A newer version? 2017 is the latest version?

**Comment by purnadika** [comment]
Yeah. you know the version like v2017.**101** and mine is like v2017.**201**

**Comment by tinohager** [comment]
On which part you found this version `v2017.201` ?

**Comment by purnadika** [comment]
that was just for illustration.
Here's screenshot of mine:
![image](https://user-images.githubusercontent.com/9426610/48968573-08af5f80-f024-11e8-9725-4683f09ccd48.png)

while what I have downloaded is
![image](https://user-images.githubusercontent.com/9426610/48968579-21b81080-f024-11e8-89ad-8d3a9ee77e32.png)

**Comment by tinohager** [comment]
Can you send me the output of `reg query "HKCR\Installer\Dependencies\,,amd64,14.0,bundle"`

![image](https://user-images.githubusercontent.com/8215186/48968658-cd486d00-eff2-11e8-8851-50e6b5c5125a.png)

**Comment by purnadika** [comment]
![image](https://user-images.githubusercontent.com/9426610/48968686-82941880-f025-11e8-8a31-0c3384775b51.png)

**Comment by tinohager** [comment]
can you check this command req query "HKCR\Installer\Dependencies\VC,redist.x64,amd64,14.16,bundle"

**Comment by purnadika** [comment]
![image](https://user-images.githubusercontent.com/9426610/48968839-e0296480-f027-11e8-92c4-de2b990fbb9c.png)

**Comment by tinohager** [comment]
I have push a new nuget package, i think now its works 2.2.4 (available in few minutes)
https://www.nuget.org/packages/Alturos.Yolo/2.2.4

If it works for you now, I would be happy to receive a star for the project

**Comment by purnadika** [comment]
great! works like charm

**Comment by Nav055ec** [comment]
@purnadika : I am facing same problem, I am new to this. Can you please share the steps how did you fix this?
